### PR TITLE
Changing the order from ascending to descending

### PR DESCRIPTION
**Pull request by juggernautsei**
Fixes #

### CODE REVIEW

**Comment by juggernautsei** [comment]
bump

**Comment by juggernautsei** [comment]
here we are again. Keep at it right?

File: interface/billing/billing_tracker.php [hunk 1]
```diff
@@ -94,7 +94,7 @@
                     { "data": "created_at" },
                     { "data": "updated_at" },
                 ],
-                "order": [[4, 'asc']] // Order by 'Date Created' with newest first
+                "order": [[4, 'desc']] // Order by 'Date Created' with newest first
```
Comment (bradymiller) [inline interface/billing/billing_tracker.php:97]: does the comment need to change?

Comment (juggernautsei) [reply interface/billing/billing_tracker.php:97]: Yes it does.

**Comment by juggernautsei** [comment]
This is already in the codebase.
https://github.com/openemr/openemr/blob/52773322af5a42b2622f0ce285433be6a1247323/interface/billing/billing_tracker.php#L97

**Pull request closed without merging**